src/docs/asciidoc/reference/development/custom-cloud-services-several.adoc

```diff
@@ -1,2 +1,150 @@
 :relative-path: ../../
 include::{docdir}/variables.adoc[]
+
+== Develop a cloud service that is not exposed
+
+During the development of your _custom cloud services_ you may want to not expose some of them.
+In this case, the affected _custom cloud services_ are not reachable by the client.
+
+.Example of private _custom cloud service_
+[source, javascript, linenums]
+----
+import { Injectable } from '@zetapush/core';
+
+class MyTimeManager {                                                     // <1>
+  getCurrentTime() {                                                      // <2>
+    return Date.now();
+  }
+}
+
+@Injectable()                                                             // <3>
+export default class {
+  constructor(private timeManager: MyTimeManager) {}                      // <4>
+
+  hello() {                                                               // <5>
+    return `Hello from Worker at ${this.timeManager.getCurrentTime()}`;
+  }
+}
+----
+<1> Our private _custom cloud service_ (no `export` keyword)
+<2> `getCurrentTime()` is not reachable by the client (front part of your application)
+<3> We specify that we want to use the dependency injection in our class
+<4> We ask the dependency injection to have an instance of `MyTimeManager`
+<5> `hello()` is the only method reachable by the client (front part of your application)
+
+[#expose_several_cloud_services]
+== Expose several cloud services
+
+_ZetaPush_ provides the possibility to expose several _custom cloud services_ if you want.
+To do this, you need to use the _namespaces_.
```

Comment:
We need to explain why it is necessary. We should say that several cloud services may have same cloud function names so we provide a simple way to differentiate them. Namespace is the technical word we use.